app/models/UsersRepository.scala

```diff
@@ -20,26 +23,27 @@ case class UserInfo(email: String,
                     admin: Boolean,
                     _id: BSONObjectID = BSONObjectID.generate)
 
+case class UpdatedUserInfo(email: String, active: Boolean, password: Option[String])
+
 object UserJsonFormats {
 
   import play.api.libs.json.Json
 
+  val pageSize = 10
+
   implicit val userFormat = Json.format[UserInfo]
 }
 
 class UsersRepository @Inject()(reactiveMongoApi: ReactiveMongoApi) {
 
   import play.modules.reactivemongo.json._
 
-  protected def collection: Future[JSONCollection] = reactiveMongoApi.database.map(_.collection[JSONCollection]("users"))
-
-  def getByEmail(email: String)(implicit ex: ExecutionContext): Future[List[UserInfo]] = {
+  def getByEmail(email: String)(implicit ex: ExecutionContext): Future[Option[UserInfo]] = {
```

Comment:
This change has made headOption throughout the codebase redundant from wherever getByEmail is invoked, please make sure that redundancy is removed.